### ISSUE TITLE
Parallel Message Sending

### ISSUE THREAD

**Issue by robmoffat** [finos/symphony-java-toolkit#108]
### Description of Problem:
Chat Workflow is very slow

### Potential Solutions:
Allow it to send messages back to symphony in parallel.

**Comment by robmoffat** [comment]
won't fix - there are other ways to parallelize this if you need to, left up to the user.
